### DISCUSSION TITLE
Do I need a bluetooth dongle?

### DISCUSSION THREAD

**Discussion by AxeMaster1** [soundshed/soundshed-app#64] [General]
I just received my positive grid spark 40 amp, I don't own a smartphone so I am using the soundshed software on my laptop to access the 10000 or so positive grid tones from the amp site and what not.

The soundshed software is asking me to pair my amp via bluetooth to my laptop in order to have the devices "talk" to each other, since apparently my old laptop doesnt have a bluetooth adapter I presume the only way I am going to get my amp "connected" to the soundshed software on my laptop is to get a bluetooh dongle for the laptop, is this correct thinking?

Thanks
Terry

**Comment by webprofusion-chrisc** [comment]
Yes, if your laptop doesn't have bluetooth built in you would need to add it via a USB bluetooth adapter. The other thing to be aware of is that this is all basically working via Google Chrome (whether you use the installed version or the web version) and that seems to only work with 64-bit operating versions of windows (I'm assuming your laptop is not a macbook and is running Windows). So if it's too old you may find it still doesn't work.

**Reply by AxeMaster1** [reply]
Thanks for the reply, my bluetooth dongle is on the way from amazon, lol.

I am running the latest version of google chrome on a windows 10 64 bit system, so  I am good to go.

Terry
